lib/setup.ex

```diff
@@ -28,29 +32,55 @@ defmodule Setup do
   end
 
   def player_vs_player do
-    player1_name = set_player_name(1)
-    player2_name = set_player_name(2)
-    player1 = Player.setup(player1_name, :x, :human)
-    player2 = Player.setup(player2_name, :o, :human)
+    player1 = set_human_player(1, :x)
+    player2 = set_human_player(2, :o)
     {player1, player2}
   end
 
-  def player_vs_comp do
-    player1_name = set_player_name(1)
-    player2_name = set_player_name(2)
-    player1 = Player.setup(player1_name, :x, :human)
-    player2 = Player.setup(player2_name, :o, :computer)
+  def player_vs_computer do
+    player1 = set_human_player(1, :x)
+    player2 = set_computer_player(2, :o)
     {player1, player2}
   end
 
-  def comp_vs_comp do
-    player1_name = set_player_name(1)
-    player2_name = set_player_name(2)
-    player1 = Player.setup(player1_name, :x, :computer)
-    player2 = Player.setup(player2_name, :o, :computer)
+  def computer_vs_computer do
+    player1 = set_computer_player(1, :x)
+    player2 = set_computer_player(2, :o)
     {player1, player2}
   end
 
+  def set_human_player(player_number, token) do
+    human_name = set_player_name(player_number)
+    human_name
+    |> Player.human(token)
```

Comment:
I think you can simplify this:

player_number
|> set_player_name
|> Player.human(token)